recipes/mpdecimal/config.yml

```diff
@@ -1,4 +1,6 @@
 versions:
+  "4.0.0":
+    folder: "2.5.x"
```

Comment:
Looking good! But it seems a little bit weird to have the `4.x.x` versions inside the `2.5.x` folder...
Maybe if changes aren't that big, we could consider renaming the `2.5.x` folder to `all`

Reply:
Do you dare to do it?!

Reply:
Ah, yes I can :)

Reply:
Done!